### ISSUE TITLE
Conditional directional arrows

### ISSUE THREAD

**Issue by Flashfyre** [vasturiano/force-graph#100]
I would like to draw directional arrows based on my own condition, not whether there's a link both ways or not, especially since I want to use DAG and my arrowless connections are assumed to be two-way. Is it possible that something could be added like in your dash odd links example?

**Comment by vasturiano** [comment]
@Flashfyre yes, just like in the example you mentioned, you can use the method `linkCanvasObject` to draw your own links in whichever way you wish, including drawing arrows to follow the logic you desire. You just have to write a bit of `canvas` painting code.
